README.md

```diff
@@ -248,6 +248,7 @@ the Design of Network-based Software Architectures](https://www.ics.uci.edu/~fie
 * [admin-on-rest](https://github.com/marmelab/admin-on-rest) - Add a ReactJS admin GUI to any RESTful API.
 * [swagger-codegen](https://github.com/swagger-api/swagger-codegen) - Auto generation of client libraries or server stubs given an OpenAPI specification (formerly known as the Swagger Specification).
 * [Lumber](https://github.com/ForestAdmin/lumber) - Generate the admin interface of your application.
+* [oas-generator](https://github.com/isa-group/oas-generator) - NodeJS RESTfull APIs scaffolding based OpenAPI 3.x specs using oas-tools and express.
```

Comment:
I fail to understand why one would want to generate a server based on an OpenAPI spec. If the spec exists, it means that a server is already there, or am I missing something?

Reply:
This is called 'spec-driven development'. Instead of creating a server, developers write the OpenAPI spec file. Then, they run a server scaffolding tool using that OAS file, so that the server is automatically generated with mock responses. Finally, they only need to change those responses using their own business logic. The swagger-codegen package, which is already present in your readme file, serves the same purpose.